src/main/resources/com/sap/oss/phosphor/fosstars/model/rating/oss/OssArtifactSecurityRatingThresholds.json

```diff
@@ -0,0 +1,5 @@
+{
+  "good": 5.635416666666667,
+  "moderate": 4.523020833333334,
+  "unclear": 8.0
```

Comment:
I see the numbers are very precise. How did you calculate them?

Reply:
I used the thresholds from the `OssSecurityRating` as IMHO the thresholds should be the same.
I also thought about reading the `OssSecurityRatingThresholds.json` instead of creating a new one, but was not sure about this.
What do you think?

Reply:
The thresholds for the open source project security ratings are calculated based on data for a sub-set of well-known open source projects. The data don't contain info about versions. Even though the new rating contains the open source project security ratings, I am not sure if the thresholds should be the same. Could you please set the thresholds manually? Then, you can check if it makes sense to re-use the thresholds for the open source project security rating, if there is a better procedure to calculate the thresholds, etc.

Reply:
Have now set it manually.
As the expected `OssArtifactSecurityRating` scores are similar to the `OssSecurityRating` I used similar values for the threshold.
Hope this if fine.

```
{
  "good": 5.5,
  "moderate": 4.5,
  "unclear": 8.0
}
```

Reply:
Unfortunately, I don't know whether it is fine or not. We will need to figure out what the thresholds should look like. For now, it  looks okay.

Reply:
Okay.

As mentioned in another comment the artifact security score uses the project security score as base and increase it slightly if the artifact version security score is above 9 and decrease it if it is below 9.
Hence, the expected `OssArtifactSecurityRating` scores are similar to the `OssSecurityRating`.

Reply:
Okay. If you believe it should work like this, I would suggest to open an issue for this. I am not sure that reading the thresholds from a JSON file is the best way, I would take the `Thresholds` object from the instance of `OssSecurityRating`. Plus, we can get rid of a duplicate `Thresholds` class.